lib/auth.coffee

```diff
@@ -251,10 +272,12 @@ getAuth = (deps, opts) ->
 	# });
 	###
 	exports.getUserId = (callback) ->
-		token.getUserId().then (id) ->
-			throw new errors.ResinNotLoggedIn() if not id?
-			return id
-		.asCallback(callback)
+		exports.getUserDetails()
+			.then (body) ->
```

Comment:
I would prefer a var name like `userDetails` instead of body, but an even better way to do this could be with bluebird's [`.get('id')`](http://devdocs.io/bluebird/api/get).

Reply:
NIT: Most chained method calls in the SDK have the same indentation as their previous statement. Please reduce the indentation by 1 level.